server/web/src/components/dashboard/latency-harness-panel.tsx

```
@@ -0,0 +1,841 @@
+      {/* Start New Run */}
+      <Card>
+        <CardHeader>
+          <CardTitle>
+            <Zap className="w-5 h-5" />
+            Start New Test Run
+          </CardTitle>
+        </CardHeader>
+        <CardContent>
+          <div className="flex flex-wrap gap-4 items-end">
+            <div className="flex-1 min-w-[200px]">
+              <label className="block text-sm font-medium text-slate-400 mb-2">Test Suite</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)